sw/device/silicon_creator/mask_rom/mask_rom_start.S

```diff
@@ -185,6 +186,17 @@ _mask_rom_start_boot:
   andi t0, t0, 0x1
   beqz t0, .L_ast_done_loop
 
+  // Check if jittery clock should be enabled.
+  li   a0, (TOP_EARLGREY_OTP_CTRL_CORE_BASE_ADDR + \
+            OTP_CTRL_SW_CFG_WINDOW_REG_OFFSET)
+  lw   t0, OTP_CTRL_PARAM_CREATOR_SW_CFG_JITTER_EN_OFFSET(a0)
```

Comment:
@alphan i think you might want to do the same thing as what the proposed HW change is doing.
Ie, if it's NOT false, then just copy over.  That will make sure even a glitch on the read, unless it glitches to false exactly.